src/pages/index.tsx

```diff
@@ -62,8 +62,8 @@ export default function Home(): JSX.Element {
 
   return (
     <Layout
-      title={`Hello from ${siteConfig.title}`}
-      description="Description will go into a meta tag in <head />"
+      title="Sequelize - Feature-rich ORM for modern TypeScript & JavaScript"
```

Comment:
The title is currently "Hello from Sequelize | Sequelize". Hence I think we should remove the "Sequelize -" part from the title and have it "Feature-rich ORM for modern TypeScript & JavaScript | Sequelize" as a result?!

Reply:
I've put it in the front for SEO - so the title starts with 'Sequelize' in search results for the homepage

I wouldn't be against removing ` | Sequelize` from the homepage title instead

Reply:
Do we know how? Can we configure it to always have "Sequelize | <rest here>"?

Reply:
Docusaurus uses `react-helmet(-async?)`, it can be done with:
```
<Helmet titleTemplate="%s">
```

I'll give it a try, see if it works

Reply:
fixed